README.md

```diff
@@ -1,12 +1,97 @@
 certificate-transparency
 ========================
 
+#Auditing for TLS certificates#
+
 [![Build Status](https://travis-ci.org/google/certificate-transparency.svg?branch=master)](https://travis-ci.org/google/certificate-transparency)
 
-Auditing for TLS certificates.
 
-## Quickstart on Ubuntu ##
-The following steps will checkout the code and build it on a clean Ubuntu 14.04 LTS installation.  It has also been tested on an Ubuntu 15.04 installation.
+## Build With GClient ##
+
+This is now the recommended method for all supported platforms. It gives you 
+a reproducible build and avoids the need to build some dependencies manually.
+
+Known to work on FreeBSD 10, OS X (10.10) [tested with XCode + brew installation
+of deps listed below], and Ubuntu 14.04. Tested on Fedora 22 but may require
+manual override of compiler options as documented below. Tested on CentOS 7
+with similar caveats.
+
+### Install Dependencies ###
+
+Depending on which platform you have the exact packages required will vary.
+The following tools must be available for the GClient build to succeed:
+
+ - A working C++11 compiler.
+
+ - autoconf/automake etc.
+ - cmake
+ - git
+ - GNU make
+ - libtool
+ - shtool
+ - Tcl
+ - pkgconf
+ - python27
+ - [depot_tools](https://www.chromium.org/developers/how-tos/install-depot-tools)
+
+Note: On Fedora, CentOS and possibly others the libunwind-devel package must 
```

Comment:
I just tested on CentoOS and it built successfully. Removed the note as obsolete.